src/System.CommandLine.Tests/SuggestDirectiveTests.cs

```diff
@@ -273,5 +277,149 @@ public async Task It_does_not_repeat_suggestion_for_already_specified_bool_optio
                    .Should()
                    .NotContain("--bool-option");
         }
+
+        [Fact]
+        public async Task Deafault_Enum_suggestions_match_Enum_member_Names_Even_with_custom_parser() {
```

Comment:
These tests can be a little more focused and specific by moving them to `SuggestionTests` and following the approach there, which is to call `GetSuggestions` directly. No need to assert on the console output.